packages/features/bookings/lib/handleNewBooking/findBookingQuery.ts

```diff
@@ -0,0 +1,48 @@
+import prisma from "@calcom/prisma";
+
+export const findBookingQuery = async (bookingId: number) => {
```

Comment:
TODO: move these queries to repository in this PR https://github.com/calcom/cal.com/pull/15447